src/IceNucleation.jl

```diff
@@ -54,4 +55,39 @@ function dust_activated_number_fraction(
     end
 end
 
+"""
+   ABIFM_J(prs, x, T)
+
+ - `prs` - set with model parameters
+ - `x` - wt% sulphuric acid / 100
+ - `T` - air temperature [K]
+
+Returns the immersion freezing nucleation rate coefficient, `J`, in cm^-2 s^-1.
+`m` and `c` constants for illite are taken from Knopf & Alpert 2013 Table 1.
+"""
+function ABIFM_J(
+    prs::APS,
+    x::FT,
+    T::FT
+) where {FT <: Real}
+
+    thermo_params = CMP.thermodynamics_params(prs)
+
+    m = 54.48075    # for illite particles
+    c = -10.66873
+
+    w_h = 1.4408 * x
+    p_sol = exp( 23.306 - 5.3465*x + 12*x*w_h - 8.19*x*w_h^2 + ( -5814 + 928.9*x - 1876.7*x*w_h )/T ) * 100 # * 100 converts mbar --> Pa
```

Comment:
Maybe we could define it as a separate function?

Reply:
I could do that on the same file, unless you want it in Thermodynamics so it's more consistent with p_sat and p_ice?

Reply:
Lets keep it in CloudMicrophysics for now. But maybe in the end it will be better to move it to TD.jl